manim/mobject/mobject.py

```diff
@@ -464,15 +506,15 @@ def add_to_back(self, *mobjects: "Mobject") -> "Mobject":
             raise ValueError("A mobject shouldn't contain itself")
 
         for mobject in mobjects:
-            if not isinstance(mobject, Mobject):
+            if not isinstance(mobject, Mobject):  # type: ignore
```

Comment:
Why is this here?

Reply:
The type annotations already say that `mobject` should be an `Mobject` so this is an unnecessary `isinstance()` and Pyright gives an error about it.

However I left the check in because not everyone uses type annotations and presumably this runtime check was helpful in the past.

Reply:
I added a comment to explain it.